src/ebonite/build/runner/docker.py

```diff
@@ -12,18 +12,10 @@ class DockerRunnerException(Exception):
 
 
 class DockerRuntimeInstance:
```

Comment:
what is the difference between this and `DockerContainer`?

Reply:
i mean, logically

Reply:
`DockerContainer` is a public data model for containers analogical to `DockerImage`. `DockerRuntimeInstance` is an internal class used to encapsulate parameters passed to `DockerRunner`.

Reply:
this is the same case i think, if we merge `RunnerBase` inside Env, there is no need for two separate "param" classes

Reply:
I eliminated `DockerRuntimeInstance` in last version of this PR.

I dislike putting logic into persistence classes as it leads to hacks like `Task.push_model` where low-level object (`Task`) creates a higher level objects (`Ebonite`) to perform requested operation. This is a circular dependency and therefore bad design. Importing classes inside method instead of on top is just a workaround for this but it doesn't make this design any better. In case of this PR I introduced a facade to runners which is used by `Ebonite` for implementation of `run_service`, `is_service_running` and `stop_service`. This way all dependencies are on way only: client => facade => concrete runner => persistence primitives.